src/client/maplecharacter.js

```diff
@@ -257,8 +258,8 @@ class MapleCharacter {
     const twoChars = /.{1,2}/g;
     const [hat, faceAcc, ...equips] = this.equips;
 
-    const hatVslot = !hat? '' : hat.info.vslot.nValue;
-    const hatParts = !hat? [] : getParts(hat).filter(isDrawable);
+    const hatVslot = !hat ? '' : hat.info.vslot.nValue;
```

Comment:
These ternaries are the only places where VS autoformat are acceptable. See the other comment about style.